tests/tween_test.py

```diff
@@ -234,6 +236,14 @@ def test_get_swagger_versions_unsupported():
     assert 'Swagger version 10.0 is not supported' in str(excinfo.value)
 
 
+def test_user_defined_formats_get_registered():
```

Comment:
Can we add at least one acceptance test? This is pretty custom functionality, so it'd be good to stay confident it's actually working end to end. Maybe a positive and negative test?